rust/pact_matching/src/models/message.rs

```diff
@@ -301,7 +298,7 @@ impl HttpPart for Message {
     self.metadata.iter().find(|(k, _)| {
       let key = k.to_ascii_lowercase();
       key == "contenttype" || key == "content-type"
-    }).map(|(_, v)| v.clone())
+    }).map(|(_, v)| v[0].as_str().unwrap_or_default().to_string())
```

Comment:
This looks strange to me. Is the intent to convert the Value to a string? There is a util function `json_to_string` which does that

Reply:
Thanks. I'll check to see other uses of that now too.

Reply:
Aha! I knew I'd  seen that pattern elsewhere :P 

![Screen Shot 2021-07-06 at 2 31 48 pm](https://user-images.githubusercontent.com/53900/124542545-e8481280-de66-11eb-8b88-70f5cc7d510b.png)

I've found 3 other places that used this same approach before and have changed. Could you please double check?

The tests pass and my complex metadata check with Pact Go also passes, but good to get a second pair of eyes on it.